tests/providerTest.js

```diff
@@ -144,6 +149,32 @@ describe('provider testing', () => {
       },
       'provider base url is returned': () => {
         return { providerBaseURL: process.env.PROVIDER_BASE_URL }
+      },
+      'resource is shared': (setup, parameters) => {
+        if (setup) {
+          const { username, password, resource, shareType, shareWith } = parameters
+          const response = shareResource(username, password, resource, shareType, shareWith)
+
+          const { status } = getOCSMeta(response)
+          /* eslint-disable-next-line */
```

Comment:
please disable only one rule that's causing error

Reply:
this is to disable the following one line,
how can I disable a specific rule?

Reply:
e.g `// eslint-disable-next-line no-useless-escape`